gdlsc.go

```diff
@@ -1,104 +1,94 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
-	"os/exec"
+	"path"
+	"path/filepath"
 	"strings"
+
+	"github.com/ryanuber/go-license"
 )
 
-// URL represents path a github repos license
-var URL = "https://api.github.com/repos/%s/license?access_token=" + os.Getenv("ACCESS_TOKEN")
+const root = "vendor/"
 
-//types for message to be printed
-type message struct {
-	Dependency string
-	License    string
-	Link       string
-}
+var files []string
+var rootPath string
+var noLicense = map[string]bool{}
 
-// options for response from github
-type response struct {
-	Html_URL string
-	License  license
+// Response contains license and file info
+type Response struct {
```

Comment:
This struct could be private (i.e. lowercase names). Also not a big deal so feel free to leave it as is, but just as a general practice only things that need to be public should be.